src/PrestaShopBundle/Resources/config/services/adapter/quick_access.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  _defaults:
+    public: false
+    bind:
+      $connection: '@doctrine.dbal.default_connection'
```

Comment:
Doesn't it  work without this line ? In QuickAccessRepository the param $connection is hint typed, then autowiring should work fine. In any case I think so.